### PR TITLE
Creating and downloading new log file

### PR DESCRIPTION
**Pull request by khodand**
image

### CODE REVIEW

File: www/cgi-bin/create-log.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#Copyright 2018 - 2019 Andrei Khodko
```
Comment (anastasiia-kornilova) [inline www/cgi-bin/create-log.sh]: It was created in 2019 year

File: www/cgi-bin/create-log.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License
+
+#!/bin/sh
+
+FILENAME=$(./etc/trik/log_manager.sh --all)
+FILESIZE=$(stat -c%s "$FILENAME")
+
+cat << EOF
```
Comment (anastasiia-kornilova) [inline www/cgi-bin/create-log.sh]: Add support for notifier to check on desktop version

File: www/configurator.html [hunk 3]
```diff
@@ -132,7 +132,9 @@
 
 
         </div>
-
+        <button type="button" class="btn btn-primary mr-3" @click="downloadLogs()" id="logs"
```
Comment (anastasiia-kornilova) [inline www/configurator.html]: Attach screenshot to PR, please. It is more comfortable to estimate designed solution.

File: www/js/locale.js [hunk 4]
```diff
@@ -88,6 +89,7 @@ window.translations = {
         'infoSet': 'These settings are designed for advanced users.',
         'inputShouldBe' : 'Input should be',
         'leaderIP' : 'Leader IP',
+        'logs': "Download logs",
```
Comment (anastasiia-kornilova) [inline www/js/locale.js]: Let's discuss the name of this option with @iakov. Maybe "Export logs"?

File: www/configurator.html (hunk 3 above)
Comment (khodand) [reply www/configurator.html]: image

File: www/cgi-bin/create-log.sh (hunk 2 above)
Comment (khodand) [reply www/cgi-bin/create-log.sh]: `/notifyThenKill.sh $(basename -- "$0") $$`

File: www/js/configurator.js [hunk 5]
```diff
@@ -301,7 +302,32 @@ const app = new Vue({
...
+                        app.logsPath = xhr.responseText;
+
+                        var link = document.getElementById('logsLink');
+                        var filename =`${app.logsPath}`.substr(`${app.logsPath}`.lastIndexOf('/') + 1);
+
+                        link.setAttribute('href', filename);
+                        link.setAttribute('download', filename);
+                        link.click();
+                    } else {
+                        //mb notification if load is impossible
```
Comment (khodand) [inline www/js/configurator.js]: Any ideas how to do it best? As option we can do dialog window such as on "Save" buttons.
image

File: www/configurator.html [hunk 6]
```diff
@@ -132,7 +132,11 @@
 
 
         </div>
-
+	    <a id="logsLink"></a>
+        <a href="#" class="mr-3" @click="downloadLogs()" id="logsPopover" data-toggle="popover" data-trigger="hover"
```
Comment (khodand) [inline www/configurator.html]: image

File: www/cgi-bin/create-log.sh [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License
+
+#!/bin/sh
+
+#./notifyThenKill.sh $(basename -- "$0") $$
```
Comment (khodand) [inline www/cgi-bin/create-log.sh]:
```suggestion
./notifyThenKill.sh $(basename -- "$0") $$
```

**Comment by anastasiia-kornilova** [comment]
It doesn't work on downloading.
image

File: www/js/configurator.js (hunk 5 above)
Comment (anastasiia-kornilova) [reply www/js/configurator.js]: Move this to separate issue, please

**Pull request merged**